### PR TITLE
Updated manage message queues instructions

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request
Updates to message queue information for Bulk API:
- Clarify note about starting the `async.operations.all` consumer that handles asynchronous and bulk API messages.
- Add prerequisite to install and configure RabbitMQ before using Bulk API

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.2/rest/bulk-endpoints.html
- https://devdocs.magento.com/guides/v2.3/rest/bulk-endpoints.html
- https://devdocs.magento.com/guides/v2.2/install-gde/prereq/install-rabbitmq.html
- https://devdocs.magento.com/guides/v2.2/install-gde/prereq/install-rabbitmq.html
- https://devdocs.magento.com/guides/v2.3/install-gde/prereq/install-rabbitmq.html

### CODE REVIEW

File: guides/v2.2/install-gde/prereq/install-rabbitmq.md [hunk 1]
```diff
@@ -24,8 +24,8 @@ The message queue system must be established before you install Magento. The bas
 1. Install RabbitMQ and any prerequisites.
 1. Connect RabbitMQ and Magento.
 
-{:.bs-callout .bs-callout-info}
-A basic message queue system can be implemented on {{site.data.var.ee}} using cron instead of RabbitMQ. See [Configure message queues]({{ page.baseurl }}/config-guide/mq/manage-mysql.html) for more information.
+{:.bs-callout-info}
+You can use MySQL or RabbitMQ for message queue processing. For details on setting up the message queue system, see [Message queues overview]({{ page.baseurl }}/extension-dev-guide/message-queues/message-queues.html). If you are using the Bulk API with {{ site.data.var.ee }} v2.3.3 or later, the default message queue configuration uses RabbitMQ as the message broker.
```
Comment (hostep) [inline guides/v2.2/install-gde/prereq/install-rabbitmq.md]: I wouldn't specify a Magento version here, or at least change it to 2.3.0. I think the message queue architecture was introduced somewhere in Magento 2.2 for Magento Commerce and made available for Magento Open Source with 2.3 and it probably always already supported RabbitMQ? Bulk API itself was introduced in version 2.3.0 as far as I know.

File: guides/v2.3/install-gde/prereq/install-rabbitmq.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ The message queue system must be established before you install Magento. The bas
 1. Connect RabbitMQ and Magento.
 
 {:.bs-callout .bs-callout-info"}
-A basic message queue system can be implemented using cron instead of RabbitMQ. See [Configure message queues]({{page.baseurl}}/config-guide/mq/manage-mysql.html) for more information.
+You can use MySQL or RabbitMQ for message queue processing. For details on setting up the message queue system, see [Message queues overview]({{ page.baseurl }}/extension-dev-guide/message-queues/message-queues.html). If you are using the Bulk API with {{ site.data.var.ee }} v2.3.3 or later, the message queue system configuration defaults to using RabbitMQ as the message broker.  See [Start message queue consumers]({{page.baseurl}}/config-guide/mq/manage-message-queues.html#start-message-queue-consumers) for more information.
```
Comment (keharper) [inline guides/v2.3/install-gde/prereq/install-rabbitmq.md]: See hostep's comment above.

File: guides/v2.3/rest/bulk-endpoints.md [hunk 3]
```diff
@@ -9,8 +9,8 @@ functional_areas:
 
 Bulk API endpoints differ from other REST endpoints in that they combine multiple calls of the same type into an array and execute them as a single request. The endpoint handler splits the array into individual entities and writes them as separate messages to the message queue.
 
-{:.bs-callout .bs-callout-tip}
-Use the `bin/magento queue:consumers:start async.operations.all` command to enable bulk endpoint processing.
+{:.bs-callout-info}
+Use the [`bin/magento queue:consumers:start async.operations.all`](https://devdocs.magento.com/guides/v2.3/config-guide/cli/config-cli-subcommands-queue.html) command to start the consumer that handles asynchronous and bulk API messages. Also, before using the Bulk API to process messages, you must install and configure RabbitMQ, which is the default message broker. See [RabbitMQ]({{ page.baseurl }}/install-gde/prereq/install-rabbitmq.html).
```
Comment (keharper) [inline guides/v2.3/rest/bulk-endpoints.md]: Don't hardcode the link.

File: guides/v2.2/rest/bulk-endpoints.md [hunk 4]
```diff
@@ -10,8 +10,8 @@ functional_areas:
 
 Bulk API endpoints differ from other REST endpoints in that they combine multiple calls of the same type into an array and execute them as a single request. The endpoint handler splits the array into individual entities and writes them as separate messages to the message queue.
 
-{:.bs-callout .bs-callout-tip}
-Use the `bin/magento queue:consumers:start async.operations.all` command to enable bulk endpoint processing.
+{:.bs-callout-info}
+Use the [`bin/magento queue:consumers:start async.operations.all`](https://devdocs.magento.com/guides/v2.3/config-guide/cli/config-cli-subcommands-queue.html) command to start the consumer that handles asynchronous and bulk API messages. Also, before using the Bulk API to process messages, you must install and configure RabbitMQ, which is the default message broker. See [RabbitMQ]({{ page.baseurl }}/install-gde/prereq/install-rabbitmq.html).
```
[the remark this answers is not in the document]
Comment (meker12) [reply guides/v2.2/rest/bulk-endpoints.md]: That was an oversight. Thx

Comment (nuzil) [inline guides/v2.2/rest/bulk-endpoints.md]: Please also lets use always same writing standard: Bulk API and Asynchronous (First capital). Pls check everywhere

Comment (meker12) [reply guides/v2.2/rest/bulk-endpoints.md]: > Please also lets use always same writing standard: Bulk API and Asynchronous (First capital). Pls check everywhere

You are correct. We aren't consistent in our capitalization.  However, this issue is outside scope of this PR.  Suggest opening a new PR to review and update occurrences of references to "bulk API" and "asynchronous" as needed.  When we reference "Bulk API" as a feature name makes sense to capitalize, but sometimes we have references more generic references like bulk API or asynchronous messages, which do not need to be capitalized.

Comment (nuzil) [reply guides/v2.2/rest/bulk-endpoints.md]: Agree)) but maybe we have to start from somewhere 😂

**Comment by nuzil** [comment]
Can you please sync branch with master?

File: guides/v2.2/config-guide/mq/manage-message-queues.md [hunk 5]
```diff
@@ -40,6 +40,9 @@ You can also use a process manager such as [Supervisor](http://supervisord.org/i
 *  Cron job `consumers_runner` runs all defined consumers
 *  Each consumer processes 10000 messages and then terminates
 
+{:.bs-callout-info}
+If your {{ site.data.var.ee }} store is hosted on the Cloud platform, use the [`CRON_CONSUMERS_RUNNER`]({{ page.baseurl }}/cloud/env/variables-deploy.html#cron_consumers_runner) to configure the `consumers_runner` cron job.
```
Comment (meker12) [inline guides/v2.2/config-guide/mq/manage-message-queues.md]:
```suggestion
If your {{ site.data.var.ee }} store is hosted on the Cloud platform, use the [`CRON_CONSUMERS_RUNNER`]({{ page.baseurl }}/cloud/env/variables-deploy.html#cron_consumers_runner) deploy variable to configure the `consumers_runner` cron job.
```

File: guides/v2.3/install-gde/prereq/install-rabbitmq.md [hunk 6]
```diff
@@ -21,7 +21,7 @@ The message queue system must be established before you install Magento. The bas
 1. Connect RabbitMQ and Magento.
 
 {:.bs-callout .bs-callout-info"}
-A basic message queue system can be implemented using cron instead of RabbitMQ. See [Configure message queues]({{page.baseurl}}/config-guide/mq/manage-mysql.html) for more information.
+You can use MySQL or RabbitMQ for message queue processing. For details on setting up the message queue system, see [Message queues overview]({{ page.baseurl }}/extension-dev-guide/message-queues/message-queues.html). If you are using the Bulk API with {{ site.data.var.ee }}, the message queue system configuration defaults to using RabbitMQ as the message broker.  See [Start message queue consumers]({{page.baseurl}}/config-guide/mq/manage-message-queues.html#start-message-queue-consumers) for more information.
```
Comment (meker12) [inline guides/v2.3/install-gde/prereq/install-rabbitmq.md]:
```suggestion
You can use MySQL or RabbitMQ for message queue processing. For details on setting up the message queue system, see [Message queues overview]({{ page.baseurl }}/extension-dev-guide/message-queues/message-queues.html). If you are using the Bulk API with {{ site.data.var.ee }}, the message queue system configuration defaults to using RabbitMQ as the message broker. See [Start message queue consumers]({{page.baseurl}}/config-guide/mq/manage-message-queues.html#start-message-queue-consumers) for more information.
```

**Comment by meker12** [comment]
running tests

File: guides/v2.2/rest/bulk-endpoints.md [hunk 7]
```diff
@@ -10,8 +10,8 @@ functional_areas:
 
 Bulk API endpoints differ from other REST endpoints in that they combine multiple calls of the same type into an array and execute them as a single request. The endpoint handler splits the array into individual entities and writes them as separate messages to the message queue.
 
-{:.bs-callout .bs-callout-tip}
-Use the `bin/magento queue:consumers:start async.operations.all` command to enable bulk endpoint processing.
+{:.bs-callout-info}
+Use the [`bin/magento queue:consumers:start async.operations.all`]({{ page.baseurl }}/config-guide/cli/config-cli-subcommands-queue.html) command to start the consumer that handles asynchronous and bulk API messages. Also, before using the Bulk API to process messages, you must install and configure RabbitMQ, which is the default message broker. See [RabbitMQ]({{ page.baseurl }}/install-gde/prereq/install-rabbitmq.html).
```
Comment (meker12) [inline guides/v2.2/rest/bulk-endpoints.md]:
```suggestion
Use the `bin/magento queue:consumers:start async.operations.all` command to start the consumer that handles asynchronous and bulk API messages. Also, before using the Bulk API to process messages, you must install and configure RabbitMQ, which is the default message broker. See [RabbitMQ]({{ page.baseurl }}/install-gde/prereq/install-rabbitmq.html).
```

**Comment by meker12** [comment]
running tests

**Pull request merged**